src/__tests__/__snapshots__/SearchPanel.spec.jsx.snap

```diff
@@ -0,0 +1,167 @@
+exports[`component:SearchPanel displays last updated date when present 1`] = `
+<div
+  className="search-panel">
+  <h2>
+    Search complaint data 
```

Comment:
...but no date is displayed here.  That's why the coverage is not at 100%

https://coveralls.io/builds/13474174/source?filename=src%2FSearchPanel.jsx

For some reason, in your test setup, `results.lastUpdated` is not being set like you think it is